app/src/main/java/nl/eduvpn/app/service/PreferencesService.java

```diff
@@ -378,6 +394,20 @@ public void storeSavedAuthStateList(@NonNull List<SavedAuthState> savedAuthState
         }
     }
 
+    /**
+     * Stores the instance list for a specific connection type
+     */
+    public void storeOrganizationList(OrganizationList instanceListToSave) {
+        /*** TODO
```

Comment:
cocwoc?

Reply:
Yes and no, will implement in next PR :)